### DISCUSSION TITLE
Pricing Suggestion for Smaller Projects

### DISCUSSION THREAD

**Discussion by jeremyleonardo** [supabase/supabase#7037] [Feature Requests]
### Considerations
- Supabase currently doesn't have any way to create a lot of small projects affordably (no, it's not really affordable to pay $25/project if I have a lot of small projects). 
- The other problem is, Supabase does not really have an "easy way" of self-hosting, it's a bit harder to self-host for our hobby project

### Suggestions
1. Create a cheaper "hobby tier" (with more limited capabilities if compared to pro tier)
2. A pricing model with shared limits for all of our projects. (e.g. a storage of 1GB for all of my projects)
3. An option for full PAYG mode

Is it possible to implement maybe one of them?

**Comment by michaelpumo** [comment]
It would be great to see a pricing tier that isn't fixed but contains all features with a **_pay as you scale_** model...a little like Firebase does.
https://firebase.google.com/pricing

In this way, projects that aren't using many resources pay _little_ and those using more pay _much_ more. It not only makes the system simpler but is also more fair.

What does Supabase think about implementing this? I think it would actually bring in a lot of paying customers who are stuck in that middle-ground between Free and Pro plans but I can't make that judgement as I don't have the figures.

You could even have a small basic flat-fee of say $5 p/m + scaling prices if you were concerned about more consistent and predictable revenue.

**Comment by criticalmash** [comment]
I'm also wanting to build a lot of small projects with Supabase. For most of them, I'd be the only authenticated user. In other cases, there would be a couple authenticated users and many anonymous, read-only users.

I was thinking about getting a single Pro account and then creating a separate schema for each project. but I'm not sure how I would get auth to work with multiple schemas.

It would be great if Supabase had an easy way to create new schemas and a role for accessing just that schema, then specify roles on a user-by-user basis. I would definitely pay $25/month for that.
